test/web_console/helper_test.rb

```diff
@@ -23,6 +23,14 @@ def call(env)
         def request
           Request.new(@env)
         end
+
+        def status
```

Comment:
Unfortunately, tests do not pass without the header because it's getting parsed by the logic that checks if the console has to be included.

Regarding the status, we're forced to pass it because is the second arg. We _could_ pass a nil value, but I would prefer to use some valid value (e.g. 500, 200, etc) because having incomplete or invalid test data could lead to confusion later.

Anyway, I'm open to change the invocation if you want to use another status value or just pass nil.